ogc_ws/src/yonah_rqt/src/yonah_rqt/main_window.py

```diff
@@ -73,15 +74,17 @@ def __init__(self, context):
         self.SummaryWindow = SummaryWindow(self.aircraft_list)
         self.CommandWindow = CommandWindow(self.aircraft_list)
 
+        self.CommandWindow.change_valid_ids()
         self.create_layout()
         self.shortcuts()
-        self.CommandWindow.change_valid_ids()
+
         # Subscriber lists
         rospy.Subscriber("ogc/from_despatcher/regular", RegularPayload, self.regular_payload)
         rospy.Subscriber("ogc/yonahtext", String, self.status_text)
         rospy.Subscriber("ogc/from_despatcher/ondemand", String, self.ondemand)
         rospy.Subscriber("ogc/files/conflict", String, self.syncthing)
         rospy.Subscriber("ogc/feedback_to_rqt", LinkMessage, self.feedback_message)
+        rospy.Subscriber("ogc/stuff", String, self.link_status)
```

Comment:
The topic where the link status is published to is `ogc/from_despatcher/ondemand`, not `ogc/stuff`